src/trashinfo.rs

```diff
@@ -14,10 +14,16 @@ impl Trashinfo {
             2: deletion_date.to_string(),
         }
     }
+    
     pub fn to_contents(&self) -> String {
         let pct_string = percent_encode(&self.1);
         format!("{}\nPath={}\nDeletionDate={}\n", self.0, pct_string, self.2)
     }
+
+    pub fn from_file(path: &str) -> io::Result<Trashinfo> {
+        let info = configster::parse_file(path, '\n')?;
+        Ok(Trashinfo::new(&percent_decode(&info[1].value.primary).unwrap(), &info[2].value.primary))
```

Comment:
You'll want to check to see if it's an absolute path or not. If it doesn't start with a '/'. If it doesn't start with a '/', it needs to get restored to the media root.

Reply:
> You'll want to check to see if it's an absolute path or not. If it doesn't start with a '/'. If it doesn't start with a '/', it needs to get restored to the media root.

Actually let's leave this for a different patch. I think it would be too difficult to test right now because rmwrs only writes absolute paths to the Path key.